common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/EmbeddedWebViewAuthorizationStrategy.java

```diff
@@ -69,7 +70,7 @@ public AuthorizationResult requestAuthorization(@NonNull final AuthorizationRequ
      * @throws ClientException
      */
     public EmbeddedWebViewAuthorizationStrategy(@NonNull final Activity activity,
```

Comment:
We're increasing our coupling here between `EmbeddedWebViewAuthorizationStrategy` and Microsoft's services. I recall giving the feedback that the `EmbeddedWebViewAuthorizationStrategy` really ought to accept the `WebViewClient` as a parameter and this is exactly why.

If we're ever going to add support for other IdP's (and it is our goal to do this over a long enough timespan), this will have to get refactored out.